### PR TITLE
Add-35_Search_Insert_Position

### PR DESCRIPTION
**Pull request by Rwatana**
Please check my code

### CODE REVIEW

File: 35_Search_Insert_Position.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+class Solution:
+    def searchInsert(self, nums: List[int], target: int) -> int:
+        middle = 0
+        left, right = 0, len(nums) - 1
+        if nums[0] > target:
+            return 0
+        if nums[-1] < target:
+            return len(nums)
+        while left <= right:
+            middle = (left + right) // 2
```
Comment (ryujumoto) [inline 35_Search_Insert_Position.py]: middle = (left+right) // 2

File: 35_Search_Insert_Position.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# T(logN) O(N)
```
Comment (ryujumoto) [inline 35_Search_Insert_Position.py]: #time: O()
#space: O()

**Review by ryujumoto — COMMENTED** [review]
Done.

File: 35_Search_Insert_Position.py (hunk 1 above)
Comment (Rwatana) [reply 35_Search_Insert_Position.py]: @Ryuju0911 
Let me confirm whether I have to change or not.
If I change from mine into yours, I got error on vscode.
```
	"message": "missing whitespace around arithmetic operator",
```
Which way should I choose?

**Review by ryujumoto — APPROVED** [review]

**Pull request merged**